sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/implementation/models/MapAttribution.java

```diff
@@ -1,8 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
 // Code generated by Microsoft (R) AutoRest Code Generator.
 
-package com.azure.maps.render.models;
+package com.azure.maps.render.implementation.models;
```

Comment:
All models used directly by the client (returned by the client) need to be in the public models folder.